### ISSUE TITLE
E1120 false-positive on deque.rotate() (no-value-for-parameter)

### ISSUE THREAD

**Issue by ypsah** [PyCQA/pylint#2013]
### Steps to reproduce
```python
from collections import deque

def test_deque_rotate_without_arg():
    deque().rotate()
```

### Current behavior
`No value for argument 'n' in method call (no-value-for-parameter)`

### Expected behavior
No error: as [`deque.rotate`](https://docs.python.org/3/library/collections.html#collections.deque.rotate)'s documentation states, n defaults to 1. 

### pylint --version output
No config file found, using default configuration
pylint 1.8.4, 
astroid 1.6.3
Python 3.4.5 (default, Dec 11 2017, 14:22:24) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-16)]

**Comment by PCManticore** [comment]
Thanks! Should be easy to fix by adding `n=1` to `brain_collections` in astroid.
